content/en/logs/explorer/watchdog_insights.md

```diff
@@ -12,68 +12,68 @@ further_reading:
     - link: 'logs/explorer/#list-of-logs'
       tag: 'Documentation'
       text: 'The list view of logs'
+      
 ---
 
 ## Overview
 
-With Watchdog Insights, find your way faster to the root cause of problems with contextual insights you receive as you explore logs. Watchdog Insights complements your expertise and instincts, giving nudges toward where you could focus your troubleshooting attention. 
+Datadog Log Management offers Watchdog Insights to help you navigate to the root cause of problems with contextual insights in the Log Explorer. Watchdog Insights complement your expertise and instincts by recommending outliers and potential performance bottlenecks impacting a subset of users.
 
 <div class="alert alert-warning">
-Watchdog Insights for the Log Explorer is a beta feature, rolling out to customers using Log Management. If you have feedback, contact <a href="https://docs.datadoghq.com/help">Datadog support</a>.
+Watchdog Insights for the Log Explorer is in beta. Access to this feature is provisioned to customers using Log Management. If you have feedback, contact [Datadog support][3].
```

Comment:
Link is broken, needs to be inside <a> tags `<a href=""></a>`

Reply:
```suggestion
Watchdog Insights for the Log Explorer is in beta. Access to this feature is provisioned to customers using Log Management. If you have feedback, contact <a href="https://docs.datadoghq.com/help">Datadog support</a>.
```